src/impl/locale.js

```diff
@@ -167,7 +175,7 @@ function supportsFastNumbers(loc) {
       loc.numberingSystem === "latn" ||
       !loc.locale ||
       loc.locale.startsWith("en") ||
-      new Intl.DateTimeFormat(loc.intl).resolvedOptions().numberingSystem === "latn"
+      getCachedIntResolvedOptions(loc.locale).numberingSystem === "latn"
```

Comment:
Should this be `loc.intl`?

Reply:
It looks a bit strange, because in my last commit I reverted this change. getCachedIntResolvedOptions function is used only in supportsFastNumbers and isEnglish .

See the code in master: https://github.com/moment/luxon/blob/master/src/impl/locale.js

Reply:
Ops, I see your pointed code better now, sorry and you're right it should be intl.

Reply:
https://github.com/moment/luxon/pull/1654